frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -1011,6 +1011,73 @@ describe("scenarios > question > filter", () => {
     cy.findByText("AL").click();
     cy.button("Add filter").isVisibleInPopover();
   });
+
+  describe.skip("should be able to filter on the boolean column (metabase#16386)", () => {
+    beforeEach(() => {
+      cy.createNativeQuestion({
+        name: "16386",
+        native: {
+          query:
+            'select 0::integer as "integer", true::boolean AS "boolean" union all \nselect 1::integer as "integer", false::boolean AS "boolean" union all \nselect null as "integer", true::boolean AS "boolean" union all \nselect -1::integer as "integer", null AS "boolean"',
+        },
+        visualization_settings: {
+          "table.pivot_column": "boolean",
+          "table.cell_column": "integer",
+        },
+      }).then(({ body: { id: QUESTION_ID } }) => {
+        cy.visit(`/question/${QUESTION_ID}`);
+        cy.findByText("Explore results").click();
+      });
+    });
+
+    it("from the column popover (metabase#16386-1)", () => {
+      cy.get(".cellData")
+        .contains("boolean")
+        .click();
+
+      popover()
+        .findByText("Filter by this column")
+        .click();
+
+      // Not sure exactly what this popover will look like when this issue is fixed but it should probably have an input field
+      popover().find("input");
```

Comment:
It works similar to the other two filtering ways, but it seems like you want to figure out a better way to detect that.
![image](https://user-images.githubusercontent.com/1447303/121252689-984a5000-c8a8-11eb-884b-df0c924545ac.png)